dialogue-core/src/main/java/com/palantir/dialogue/core/RetryingChannel.java

```diff
@@ -143,12 +144,20 @@
                 .channelName(channelName)
                 .reason("qosResponse")
                 .build();
+        this.retryDueToThrowable = DialogueClientMetrics.of(metrics)
+                .requestRetry()
+                .channelName(channelName)
+                .reason("throwable")
+                .build();
```

Comment:
I also tacked on this bit of instrumentation, as I realized we missed a case!

Reply:
Nice catch, thanks!